flow/connectors/sql/query_executor.go

```diff
@@ -410,7 +410,7 @@ func toQValue(kind qvalue.QValueKind, val interface{}) (qvalue.QValue, error) {
 
 	case qvalue.QValueKindJSON:
 		vraw := val.(*interface{})
-		vstring := (*vraw).(string)
+		vstring, _ := (*vraw).(string)
```

Comment:
should this be `vstring, ok :=`

When this is not ok, should be log something? Also when is this not ok?

Reply:
It can be not ok when you try performing 
```golang
s.sfHelper.ExecuteAndProcessQuery("SELECT j:field_which_is_not_there FROM sf_table")
```
I've pushed a commit which returns an error with explanation

Reply:
Changed it to a log instead. We can't error out there in the case of null jsons